python/vyos/ifconfig/interface.py

```diff
@@ -1015,9 +1015,11 @@ def set_mirror(self):
             source_if = next(iter(self._config['is_mirror_intf']))
             config = self._config['is_mirror_intf'][source_if].get('mirror', None)
 
+        # Please don't clear 'set $?=0'. It will return the value when executing error
```

Comment:
I do not understand this comment, from my understanding this forces the return code to be 0.

Reply:
To simulate its normal return, one is no stderr, and two is its return value 0

Reply:
In fact, any set command can only result in a return value of 0, but I write this sentence just to make sense